test/cpp/end2end/grpclb_end2end_test.cc

```diff
@@ -514,11 +522,10 @@ class GrpclbEnd2endTest : public ::testing::Test {
 
   struct AddressData {
     int port;
-    bool is_balancer;
     grpc::string balancer_name;
   };
 
-  grpc_core::ServerAddressList CreateLbAddressesFromAddressDataList(
+  static grpc_core::ServerAddressList CreateLbAddressesFromAddressDataList(
```

Comment:
nit: it's may be confusing to unconditionally associate a "balancer name" arg with each one of these addresses, since `CreateLbAddressesFromAddressDataList` is used to create not only balancer addresses but also fallback addresses.

How about pass in a bool here, `lb_addresses` ?

Reply:
The balancer name arg doesn't hurt anything for backend addresses, since it won't actually get used for anything, so I'm not going to worry about this.